fe/fe-core/src/main/java/org/apache/doris/nereids/types/DataType.java

```diff
@@ -587,4 +587,8 @@ public Map<Class<? extends DataType>, Promotion<DataType>> build() {
             return (Map) ImmutableMap.copyOf(promotionMap);
         }
     }
+
+    public double length(double high, double low) {
```

Comment:
This method looks quite wired in this class, we don't just keep this logic in StatisticRange

Reply:
rangeLength maybe a better name, because this method is used by Range.length().